tests/qm-sanity-test/check_qm_with_aib.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Create AIB manifest with the correct QM copr
+USE_QM_COPR="${PACKIT_COPR_PROJECT:-rhcontainerbot/qm}"
+sed -e "s,\${QM_COPR},${USE_QM_COPR},g" check_qm_with_aib.aib.yml > qm.aib.yml
+
+AIB="automotive-image-builder build --export qcow2 qm.aib.yml qm.qcow2"
```

Comment:
Can you explain how this file only helps to understand which mpp to use? 
I do not see use_qm true or something which this repo installed should be read?

Reply:
This is the new simplified manifest format
https://gitlab.com/CentOS/automotive/src/automotive-image-builder/-/merge_requests/130

As for the repo, it is specified in `qm.aib.yml` which is produced from `check_qm_with_aib.aib.yml` intorduced here above.

Reply:
Ok the resolve added in description, info clear the picture
I think a readme section should be added here, 

https://github.com/containers/qm/tree/main/docs/devel#building-centos-autosd-and-qm-manually